sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -249,6 +251,21 @@ public ServiceBusClientBuilder clientOptions(ClientOptions clientOptions) {
         return this;
     }
 
+    /**
+     * Sets the {@link AmqpClientOptions} to be sent from the client built from this builder, enabling customization of
+     * certain properties, as well as support the addition of custom identifier information. Refer to the {@link
+     * AmqpClientOptions} documentation for more information.
+     *
+     * @param clientOptions to be set on the client.
+     *
+     * @return The updated {@link ServiceBusClientBuilder} object.
+     */
+    @Override
+    public ServiceBusClientBuilder clientOptions(AmqpClientOptions clientOptions) {
+        this.amqpClientOptions = clientOptions;
+        return this;
+    }
```

Comment:
If we don't use it in service bus we should probably throw UnsupportedOperationException until we do.

Reply:
OK, I'll change it.